doc/TOR.md

```diff
@@ -100,6 +153,41 @@ is necessary to set up a [Tor Hidden Service].
 *NOTE: In the case where only non-persistent addresses are required,
 you don't have to create the hidden service and you can skip this part.*
 
+##### Automatic persistent .onion address
+
+It is possible to generate persistent .onion addresses automatically.
+
+Add the following lines in the `/etc/tor/torrc` file
+(you might already have done this if for example you connected Bitcoin
+over Tor):
+
+````
+ControlPort 9051
+CookieAuthentication 1
+CookieAuthFileGroupReadable 1
+````
+
+Then you can use `--addr=statictor:127.0.0.1:9051` instead of
+`--announce-addr=.onionAddressV3`.
+By default V3 onion addresses are generated.
+
+Note that you have to specify a `--bind-addr` first before using
+`--addr=statictor:`.
+Generally `--bind-addr=127.0.0.1:9735` should work fine.
+
+You can also have multiple announced persistent .onion addresses
+by adding `/torblob=BLOB`, where `BLOB` is up to 64 bytes of
+text.
+You can indicate `--addr=statictor:127.0.0.1:9051/torblob=1
+--addr=statictor:127.0.0.1/torblob=2` to create two different
+.onion addresses.
+
+Finally, the default port number for the autogenerated persistent
+.onion address will be 9735, but you can change this by adding
+`/torport=9999` to change the external port for the .onion address.
```

Comment:
Finally, the default **outbound port** number, i.e if your node is behind a vpn or nat forwarder 
the local bound port will be what you define in your generic  --addr///bind-addr=IP[:PORT] line 
siggh should have used `outbound `instead of` torport `for the label
In essence one should leave that as is, since you get a bit more visible if you are the only one who uses a specific port other than 9735, but we support the option and footguns have often dual use. ;-)

Reply:
Or "external" instead?  Outgoing suggests it is only going out, but incoming connections go into it as well.